clientd3d/animate.c

```diff
@@ -27,7 +27,7 @@
 
 #include "client.h"
 
-#define ANIMATE_INTERVAL 8  // ms between background animation updates (8.3333 = 120fps)
+#define ANIMATE_INTERVAL 8 // ms interval between background animation updates
```

Comment:
Revert this line.  I believe the new comment is incorrect; this interval has nothing to do with being in the background.